api-tester/src/main/java/com/revenuecat/apitester/java/SubscriptionOptionAPI.java

```diff
@@ -12,6 +15,31 @@ static void checkSubscriptionOption(SubscriptionOption subscriptionOption) {
         List<PricingPhase> pricingPhases = subscriptionOption.getPricingPhases();
         List<String> tags = subscriptionOption.getTags();
         Boolean isBasePlan = subscriptionOption.isBasePlan();
+        String presentedOfferingId = subscriptionOption.getPresentedOfferingIdentifier();
+        PurchasingData purchasingData = subscriptionOption.getPurchasingData();
+        String id = subscriptionOption.getId();
+
+        GoogleSubscriptionOption googleOption = (GoogleSubscriptionOption) subscriptionOption;
```

Comment:
Can this be removed?

Reply:
i guess i wanted to test that it can be casted, but i don't think this would do anything at compile time anyway now that you mention it. will do